### ISSUE TITLE
Deprecation warnings in browser console

### ISSUE THREAD

**Issue by bultas** [jhnns/rewire-webpack#8]
Hi, 

Firstly, I would like to thank you for very useful module. 

I'm  using rewire with webpack bundle, but after upgrade webpack to version 1.4.13 I'm getting some warning messages in console.log

Warning 1:

> 'window.webkitStorageInfo' is deprecated. Please use 'navigator.webkitTemporaryStorage' or 'navigator.webkitPersistentStorage' instead.

Warning 2:

> 'webkitIndexedDB' is deprecated. Please use 'indexedDB' instead.

But everything seems that works properly yet. But I write better now than when it's too late.

**Comment by srph** [comment]
Is this still an issue? I'll try it out now on 1.7.*. If ever, I'll update the outcome.

**Comment by jhnns** [comment]
Yep, it will not go away soon. rewire needs to access these properties...

**Comment by srph** [comment]
Alright :smile:! But these are merely warnings and do not affect test results, right?

**Comment by jhnns** [comment]
Yep, they occur because the browser thinks that your application is accessing these deprecated properties.
